### PR TITLE
command-line-basics - Lindsay Jin

### PR DESCRIPTION
**Pull request by lindsay-jin**
Screen Shot 2024-02-28 at 10 36 36 AM
Screen Shot 2024-02-28 at 10 36 46 AM

### CODE REVIEW

File: command-line-basics/command-line-basics-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# command-line-basics-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What is a CLI?
+
+- What is a GUI?
```
Comment (shawnkost) [inline command-line-basics/command-line-basics-quiz-notes.md]: Missing a couple quiz questions here!

**Review by shawnkost — CHANGES_REQUESTED** [review]
Missing a couple quiz questions

File: command-line-basics/command-line-basics-quiz-notes.md (hunk 1 above)
Comment (lindsay-jin) [reply command-line-basics/command-line-basics-quiz-notes.md]: Updated changes!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**